py/desisim/scripts/quickquasars.py

```diff
@@ -574,14 +603,14 @@ def simulate_one_healpix(ifilename,args,model,obsconditions,decam_and_wise_filte
             ('BRICKNAME', (str,8))]
         zbest = Table(np.zeros(nqso, dtype=columns))
         zbest["CHI2"][:]      = 0.
-        zbest["Z"]            = metadata['Z']+dz_fog
+        zbest["Z"]            = metadata['Z']
         zbest["ZERR"][:]      = 0.
 
         if args.sigma_kms_zfit:
            log.info("Added zfit error with sigma {} to zbest".format(args.sigma_kms_zfit))
-           sigma_zfit=(args.sigma_kms_zfit/299792.458)*(1.+metadata['Z'])
-           zbest["Z"]+=sigma_zfit*np.random.normal(0,1,nqso)
-           zbest["ZERR"]=sigma_zfit
+           sigma_zfit=mod_cauchy(loc=0,scale=args.sigma_kms_zfit,size=nqso,cut=3000)
```

Comment:
Hi @alxogm - Do I understand correctly that sigma_zfit here is a random variable with a shift in redshift? If so, we shouldn't call it sigma, since it suggests it has to do with a RMS, not an actual random variable. May be dz_fit? Just like we use dz_fog?

Reply:
Hello @andreufont, yes you are right. I'll change that.